### DISCUSSION TITLE
Argo Event Bus `BuildStatefulSetFailed`

### DISCUSSION THREAD

**Discussion by zaviermiller** [argoproj/argo-events#2057] [Q&A]
After creating an event bus, the status is stuck at `BuildStatefulSetFailed`. Is there a way I can get the trace of whats going on when the event bus gets created? Looking at the error in the source shows a lot of different places the error could be coming from and I was hoping someone in the community had seen this before.

Thanks!

**Comment by nene95c** [comment]
Hey there,  
Same here, did you manage sort it out?

Thanks
